viewer-prototype/src/browser/trace-viewer/components/table-output-component.tsx

```diff
@@ -140,12 +143,21 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
         }
 
         columnEntries.forEach(columnHeader => {
+            const typeString: string | null = columnHeader.type;
+            let hiddenColumn = false;
+            if (typeString) {
+                const type: ColumnHeaderType = JSON.parse(typeString);
+                hiddenColumn = type.hidden;
+            }
+
             const id = this.showIndexColumn ? ++columnHeader.id : columnHeader.id;
             colIds.push(id);
             columnsArray.push({
                 headerName: columnHeader.name,
                 field: columnHeader.id.toString(),
-                width: this.props.columnWidth
+                width: this.props.columnWidth,
+                hide: hiddenColumn,
+                tooltipField: columnHeader.description
```

Comment:
The javadoc of tooltipField says this: "The field where we get the tooltip on the object", it's not the tooltip itself. Maybe you could the tooltipValueGetter instead to return this text?